src/ProjectService.Models.Dto/Models/TaskProperty.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using LT.DigitalOffice.ProjectService.Models.Dto.Enums;
+
+namespace LT.DigitalOffice.ProjectService.Models.Dto.Models
+{
+    public class TaskProperty
```

Comment:
TaskPropertyInfo

Reply:
Deleted this file, because it is not used in my functional. It will create in Misha's PR